### PR TITLE
Add upgrading yocto notes v2

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by mirzak** [comment]
Looks good

File: .ispell_english [hunk 1]
```diff
@@ -62,6 +63,7 @@ Aggarwal
 ahmad
 ahmadnassri
 ahshagheeD
+ahW
```
Comment (oleorhagen) [inline .ispell_english]: Some interesting additions to the dictionary. Perhaps prune these a little?

File: .ispell_english [hunk 2]
```diff
@@ -672,6 +688,7 @@ en
 encodeurl
 enp
 ensureNodesAreAvailable
+enterpris
```
Comment (oleorhagen) [inline .ispell_english]: Is this not a spelling error?

File: .ispell_english [hunk 3]
```diff
@@ -1985,8 +2045,10 @@ Starkov
 stat
 StateScriptRetryInterval
 StateScriptRetryIntervalSeconds
+statescriptretryintervalseconds
```
Comment (oleorhagen) [inline .ispell_english]: Already exists in capitalised form

File: .ispell_english [hunk 4]
```diff
@@ -1985,8 +2045,10 @@ Starkov
...
 StateScriptRetryTimeout
 StateScriptRetryTimeoutSeconds
+statescriptretrytimeoutseconds
```
Comment (oleorhagen) [inline .ispell_english]: same

File: .ispell_english (hunk 1 above)
Comment (lluiscampos) [reply .ispell_english]: These magic additions all came from a very long key that the spell checker split randomly in words.

File: .ispell_english (hunk 2 above)
Comment (lluiscampos) [reply .ispell_english]: It is not, it comes from the output of the docker-compose command:
```
> menderproduction_mender-redis_1                        /redis/entrypoint.sh            
> menderproduction_mender-tenantadm_1                    /usr/bin/tenantadm --confi ...  
> menderproduction_mender-useradm_1                      /usr/bin/useradm-enterpris ...
```

Comment (lluiscampos) [reply .ispell_english]: I mean, *it is* an spelling error, of course. But we have such string there and I want the spell check to pass

File: .ispell_english (hunk 4 above)
Comment (lluiscampos) [reply .ispell_english]: It is for the anchor in the links:
```
### Retry-later

on/configuration-file/configuration-options/#statescriptretryintervalseconds) before the ...
```

Comment (lluiscampos) [reply .ispell_english]: same above

File: .ispell_english (hunk 1 above)
Comment (oleorhagen) [reply .ispell_english]: Alright. Good to go then

**Pull request merged**